src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -471,4 +471,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="NoServiceEndpoint_Description" xml:space="preserve">
     <value>--no-service-endpoint does not append "api/v2/packages" to the source URL.</value>
   </data>
+  <data name="AddPkg_InteractiveDescription" xml:space="preserve">
+    <value>Allow manual user action for operations like authentication.</value>
```

Comment:
for operations which require authentication.

Reply:
This is called only from commandlinexplat. 
Actually digging into it, I'm not even sure this string is very relevant. 
It's never displayed to the user as we don't ship commandline xplat. 

I'll try to align the string with what's decided on in the other PR - https://github.com/dotnet/cli/pull/9467/files. 

There it's currently `Allow the command to block and require manual action to proceed`